contrib/pkg/installmanager/installmanager.go

```diff
@@ -553,6 +620,36 @@ func uploadAdminPassword(cd *hivev1.ClusterDeployment, m *InstallManager) (*core
 	return s, uploadSecretWithRetries(s, m)
 }
 
+func uploadConfigmapWithRetries(c *corev1.ConfigMap, m *InstallManager) error {
+	uploaded := false
+	var err error
+
+	// try 10 times
+	for i := 1; i < 11; i++ {
```

Comment:
Suggestion: use k8s.io/apimachinery/pkg/util/wait 
```
backoff := retry.DefaultBackoff
backoff.Steps = 10
backoff.Duration =  1 * time.Second
wait.ExponentialBackoff(backoff, func() error { ... })
```
If you want to make the retry time constant, substitute retry.DefaultRetry for retry.DefaultBackoff